backend/app/assets/stylesheets/spree/backend/globals/_variables.scss

```diff
@@ -206,3 +207,14 @@ $z-index-navbar-flyout:          1000 !default;
 // Sidebar
 //--------------------------------------------------------------
 $border-sidebar:                 1px solid $color-sidebar-border !default;
+
+// Media queries
+//--------------------------------------------------------------
+$desktop-width: 959px;
+$tablet-width: 768px;
+$mobile-width: 480px;
+
+$large-device: "only screen and (min-width: #{$tablet-width})";
+$desktop: "only screen and (max-width: #{$desktop-width})";
+$tablet: "only screen and (max-width: #{$tablet-width})";
+$mobile: "only screen and (max-width: #{$mobile-width})";
```

Comment:
Prefer always going up for media queries (making sure to always use min-width). This helps create a mobile first approach:

```
$tablet: "screen and (min-width: 481px)";
$desktop: "screen and (min-width: 769px)";

.thing {
  // default styles + mobile
  style: style;

  @media $tablet {
    // more things
  }

  @media $desktop {
    // more things
  }
}
```